### PR TITLE
[Docs] change tag into comment

### PR DESCRIPTION
**Pull request by congee524**

### CODE REVIEW

**Comment by congee524** [comment]
Have checked it in my own readthedocs.

File: docs/stat.py [hunk 1]
```diff
@@ -28,10 +28,11 @@ def anchor(name):
...
 
     # count papers
-    papers = set((papertype, titlecase.titlecase(paper.lower().strip()))
-                 for (papertype, paper) in re.findall(
-                     r'\n\s*\[([A-Z]+?)\]\s*\n.*?\btitle\s*=\s*{(.*?)}',
-                     content, re.DOTALL))
+    papers = set(
+        (papertype, titlecase.titlecase(paper.lower().strip()))
+        for (papertype, paper) in re.findall(
+            r'<!--\s*\[([A-Z]*?)\]\s*-->\s*\n.*?\btitle\s*=\s*{(.*?)}',
```
Comment (innerlee) [inline docs/stat.py:34]: There is another regex in this file below

**Review by innerlee — APPROVED** [review]

**Comment by innerlee** [comment]
Are there linting issues?

**Comment by congee524** [comment]
> Are there linting issues?

I think not, have checked linting after commit.

**Pull request merged**